sentry_sdk/tracing.py

```diff
@@ -609,8 +608,8 @@ def _set_initial_sampling_decision(self, sampling_context):
             if callable(options.get("traces_sampler"))
             else (
                 # default inheritance behavior
-                sampling_context["parent_sampled"]
-                if sampling_context["parent_sampled"] is not None
+                sampling_context["transaction_context"]["parent_sampled"]
+                if sampling_context["transaction_context"]["parent_sampled"] is not None
```

Comment:
```suggestion
                sampling_context["parent_sampled"]
                if sampling_context["parent_sampled"] is not None
```

It's no longer in the transaction context, so this shouldn't be changed.